python/numpy/main.py

```diff
@@ -0,0 +1,28 @@
+from weldarray import WeldArray
```

Comment:
I'd get rid of this file before merging this PR in, just to keep the repo clean.